tests/Console/ModelsCommand/AbstractModelsCommand.php

```diff
@@ -35,5 +58,45 @@ protected function getEnvironmentSetUp($app)
             'database' => ':memory:',
             'prefix' => '',
         ]);
+
+        // Load the Models from the Test dir
+        $config->set('ide-helper.model_locations', [
+            dirname((new \ReflectionClass(static::class))->getFileName()) . '/Models',
+        ]);
+
+        // Don't override integer -> int for tests
+        $config->set('ide-helper.type_overrides', []);
```

Comment:
Because the defaults convert integer to int, the tests would fail with default config.